AHBC_MIDTERM_2019_JULY_TEAMROCKET/PaymentType.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
+{
+    class PaymentType: ICash, ICard, ICheck
+    {
+       
```

Comment:
![bitmoji](https://render.bitstrips.com/v2/cpanel/7e3bc453-64fb-4a34-bcaa-481724e5d3d9-a742c4ed-2deb-4057-a79b-4222ab646066-v1.png?transparent=1&palette=1&width=246)

Maybe with the payments, we have them be classes that interface the their respective interfaces, Than this be a abstract payment class that those classes inherit from where you set abstract methods. We can talk more about it live,

Reply:
I agree cash, credit card, and check should all be their own classes. We could still have a payment class that does all the calculating for each respective method of payment